panel/configuration.md

```diff
@@ -1,8 +1,39 @@
 # Environment Configuration
 
 [[toc]]
+## Overview
+Pterodactyls environment settings are configured and stored in the `.env` file located in your installations root directory ex. /var/www/pterodactyl. Some of the settings are also chached in the database and thoose will by default override the setting in `.env`. To change this behaviour you can edit `.env` and change the setting `APP_ENVIRONMENT_ONLY=false` to `APP_ENVIRONMENT_ONLY=true`.
+This change takes imediate effect and should be reflected on a reload of the Panel in your browser.
+Once you have resolved whatever issues you might be having you should change it back to `APP_ENVIRONMENT_ONLY=false` so that the Panel can manage it's settings in the database.
 
-## Reverse Proxy Setup
+### Manually editing the database
```

Comment:
Not sure if we want to advertise this. If we include it in the docs, it should be at the end to avoid suggesting this is recommended.